runtime/syntax/openscad.yaml

```diff
@@ -0,0 +1,48 @@
+filetype: openscad
+
+detect:
+    filename: "\\.scad$"
+
+rules:
+    - statement: "\\b(module|function|include|use|let|for|if|else|true|false)\\b"
+    - statement: "\\b(child|each|assign)\\b"
+    - statement: "\\b(intersection_for|import|export)\\b"
+
+    - function: "\\b(circle|square|polygon|cube|sphere|cylinder|polyhedron|translate|rotate|scale|resize|mirror|multmatrix|color|offset|minkowski|hull|projection|surface|render|linear_extrude|rotate_extrude|dxf_linear_extrude|dxf_rotate_extrude)\\b"
+
+    - constant: "\\b(pi|undef)\\b"
+
+    - identifier: "\\b($fn|$fa|$fs|$t|$children)\\b"
```

Comment:
I am not familiar with OpenSCAD syntax but `$` in regex marks the end of line so this is probably not what you want.

Reply:
Interestingly, it works properly without escaping the $ symbol.  Perhaps it only sees it as "end of string" if it can be at the end of a string.  I will test with it escaped and see if it still works with the change.

Reply:
Whoops, I was testing with my file, which does escape the $ symbol.

I withdraw my assertion that it works without escaping.

I got confused as to which syntax file the comment related to.